components/script/dom/servoparser/async_html.rs

```diff
@@ -334,8 +334,7 @@ impl Tokenizer {
                 .expect("Unexpected channel panic in main thread.")
             {
                 ToTokenizerMsg::ProcessOperation(parse_op) => self.process_operation(parse_op),
-                ToTokenizerMsg::End => return,
-                _ => unreachable!(),
+                _ => return,
```

Comment:
`continue` is the right one, because you want to loop to continue processing `ProcessOperation` and only return with `End`.

Reply:
Indeed